src/utils/neynar/neynar.ts

```diff
@@ -0,0 +1,44 @@
+import { version } from '../../version';
+import { FetchError } from './exceptions/FetchError';
+import { NEYNAR_DEFAULT_API_KEY } from './frame/neynarFrameFunctions';
+
+export async function fetchDataFromNeynar(url: string, apiKey: string = NEYNAR_DEFAULT_API_KEY) {
```

Comment:
Split each function in their file.
It will help organize them better and over time it's easier to mantain.

The reason we want files with the same name of the function, is that when you are looking around the folder it's intuitive to see who does what and where.

Those litle adjustemnts make an OSS library easier to mantain and understand from the outside.

Reply:
On the naming you can keep this as `getData` and `postData`. So it's easy to tell which is which, for a second I read `fetch` and I thought was a funciton that does both get and post.